package.json

```diff
@@ -3,47 +3,51 @@
   "version": "0.0.1",
   "description": "Brave Variations",
   "scripts": {
-    "test": "jest --config src/jest.config.js",
-    "lint": "eslint src --ext ts,tsx",
-    "tracker": "node src/finch_tracker/build/finch_tracker/main.js",
-    "clean": "rm -rf ./src/proto/generated/proto_bundle* src/web/public/bundle/ src/finch_tracker/build/",
+    "build": "npm run build:proto && npm run build:web && npm run build:tracker",
     "build:proto": "pbjs -t static-module --keep-case ./src/proto/variations_seed.proto ./src/proto/study.proto -o src/proto/generated/proto_bundle.js && pbts -o src/proto/generated/proto_bundle.d.ts src/proto/generated/proto_bundle.js",
     "build:tracker": "tsc -p src/finch_tracker/tsconfig.json",
     "build:web": "webpack -c src/web/webpack.config.js --mode=production",
-    "build": "npm run build:proto && npm run build:web && npm run build:tracker",
-    "serve:dev": "webpack serve -c src/web/webpack.config.js --hot --mode=development"
+    "clean": "tsx src/scripts/clean.ts",
+    "lint": "tsx src/scripts/lint.ts",
+    "serve:dev": "webpack serve -c src/web/webpack.config.js --hot --mode=development",
+    "test": "jest --config=src/jest.config.js",
+    "tracker": "node src/finch_tracker/build/finch_tracker/main.js",
+    "typecheck": "tsc --noEmit"
```

Comment:
Will this be run as a blocker for merging? Otherwise there is no type-checking for the node code.